### PR TITLE
OS specific FF download pages should always return the same build (Fixes #12192)

### LINKED ISSUE #12192: OS specific download page always downloads Firefox for current OS
## Description

Noticed that the OS specific download pages always download Firefox for Linux (on my Linux machine).

### Steps to reproduce

1. Navigate to one of: https://www.mozilla.org/en-US/firefox/windows/ https://www.mozilla.org/en-US/firefox/browsers/windows-64-bit/ https://www.mozilla.org/en-US/firefox/mac/ https://www.mozilla.org/en-US/firefox/browsers/chromebook/
2. Click "Download Now"

### Expected result

The OS specific file should download. 

### Actual result

I always get the Linux version. 

### Environment

Firefox Nightly on Linux.

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

- Updates OS specific download pages so that a single OS build is returned, as opposed to the current behavior where the button will return a link to whichever build matches your current OS.
- I also removed the nav CTA button on these pages to make things less confusing.

## Issue / Bugzilla link

- https://github.com/mozilla/bedrock/issues/12192
- https://bugzilla.mozilla.org/show_bug.cgi?id=1574162

## Testing

Demo: 

https://www-demo1.allizom.org/en-US/firefox/windows/

Local:

- http://localhost:8000/en-US/firefox/windows/
- http://localhost:8000/en-US/firefox/mac/
- http://localhost:8000/en-US/firefox/linux/
- http://localhost:8000/en-US/firefox/browsers/windows-64-bit/

### CODE REVIEW

**Comment by alexgibson** [comment]
Successful test run: https://github.com/mozilla/bedrock/actions/runs/6457156863

File: media/css/firefox/browsers/windows-64-bit.scss [hunk 1]
```diff
@@ -51,6 +51,10 @@ $image-path: '/media/protocol/img';
             margin-bottom: $spacing-md;
         }
     }
+
+    .download-firefox {
```
Comment (reemhamz) [inline media/css/firefox/browsers/windows-64-bit.scss:55]: **Nit:** The button needs some horizontal margins on smaller screens

Screenshot 2023-10-11 at 1 43 14 PM

File: bedrock/firefox/templates/firefox/browsers/windows-64-bit.html [hunk 2]
```diff
@@ -69,18 +78,12 @@ <h2>{{ ftl('windows-64-bit-how-do-you-get-64') }}</h2>
...
 {% endcall %}
 
 {% endblock %}
 
-{% block sticky_promo %}
-  {% with non_fx_only=True %}
-    {% include '/includes/sticky-promo.html' %}
-  {% endwith %}
-{% endblock %}
+{% block sticky_promo %}{% endblock %}
```
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/browsers/windows-64-bit.html:87]: **Suggestion:** Maybe remove this `sticky_promo` block if we don't have one on the page?

File: bedrock/firefox/templates/firefox/browsers/windows-64-bit.html [hunk 3]
```diff
@@ -69,18 +78,12 @@ <h2>{{ ftl('windows-64-bit-how-do-you-get-64') }}</h2>
...
 
-{% block js %}
-  {{ js_bundle('sticky_promo') }}
-{% endblock %}
+{% block js %}{% endblock %}
```
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/browsers/windows-64-bit.html:89]: **Suggestion:** Same here, we can remove this block?

**Review by reemhamz — APPROVED** [review]
Just a few nits to easily fix up!
I tested the download buttons on all the changed pages and all looks and works great and as expected. Great job!

r+wc 🪭

**Comment by alexgibson** [comment]
@reemhamz updated, thanks for the review!

**Pull request merged**